deps/rabbit/src/rabbit_stream_coordinator.erl

```diff
@@ -1900,7 +1901,7 @@ eval_replica(#{index := Idx} = Meta,
     case Current of
         undefined ->
             Args = Meta#{epoch => Epoch, node => Node},
-            Actions = [{aux, {delete_member, StreamId, Args, Conf}} |
+            Actions = [{aux, {delete_member, StreamId, Args, make_writer_conf(Conf)}} |
```

Comment:
because of the different supervisor setting using the write conf for deletes means a running replica may not actually be deleted as it would try the wrong supervisor. I think we need to run all members under the osiris_server_sup